baus.py

```diff
@@ -388,6 +389,9 @@ def get_simulation_visualization_models():
     else:
         raise "Invalid mode"
 
+print('***Copying run_setup.yaml to output directory')
+with open(os.path.join(orca.get_injectable("outputs_dir"), "run_setup.yaml"), 'w') as f:
```

Comment:
I think it is useful to have the copies be first class citizens that could be copied and used later as needed - and the comments and all that help with that. I would for that reason do a carbon copy instead of the re-mint.

Reply:
Okay will do. Do you remember if the package you used required an install? Just asking because if it was part of the existing dependencies I'll just replicate the code.

Reply:
package is just `shutils`

Reply:
It looks like that package is installed as part of the model requirements already so I didn't need to add it- yay!- I modified this code.